modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -1620,6 +1620,22 @@ CV_EXPORTS_W void blur( InputArray src, OutputArray dst,
                         Size ksize, Point anchor = Point(-1,-1),
                         int borderType = BORDER_DEFAULT );
 
+/** @brief Blurs an image using the StackBlur.
+The function applies and StackBlur to an image.
```

Comment:
Documentation formatting is broken.

New empty line is required after `@brief`.
Or `@details`.

Reply:
Will be fixed at new path.